doc/man7/RAND_DRBG.pod

```diff
@@ -144,10 +144,12 @@ together and are being used.
     +------------------+      +------------------------------------+
 
 
-
-The method L<RAND_DRBG_bytes(3)> is a convenience method wrapping the
-L<RAND_DRBG_generate(3)> function, which serves the actual request for
-random data.
```

Comment:
At [some time in the past](https://github.com/openssl/openssl/pull/5629/commits/e7516e80920b494f81245e22faca9f3f198deb08#diff-3c1c65b78c268493e63ae690dbadb753L1), this line served as explanation for the caption `<ssl->drbg> ==> RAND_DRBG_bytes(ssl->drbg, ...)` in the image above. Now since that caption disappeared together with the per-ssl DRBG, this single line was orphaned and appeared to be a bit misplaced.